cosmos/mongo.go

```diff
@@ -0,0 +1,55 @@
+// Copyright (c) Microsoft and contributors.  All rights reserved.
+//
+// This source code is licensed under the MIT license found in the
+// LICENSE file in the root directory of this source tree.
+
+package cosmos
+
+import (
+	"github.com/globalsign/mgo"
+	"github.com/globalsign/mgo/bson"
+)
+
+// GetCollection gets a mongoDB collection
+func GetCollection(session *mgo.Session, database, collectionName string) (*mgo.Collection, error) {
```

Comment:
Seems like the error return value is unnecessary.

Reply:
D: Fixed